### PR TITLE
Convert `get-ad-iframe` to TypeScript

### PR DESCRIPTION
**Pull request by arelra**
## What does this change?

Convert `get-ad-iframe` to TypeScript

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/dfp/get-ad-iframe.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+		// We need to be sure only to select the ad content frame.
+		const contentFrame = adSlot.querySelector<HTMLIFrameElement>(
+			'iframe:not([id*="__hidden__"])',
+		);
+
+		if (!contentFrame) {
+			reject();
+		} else if (
+			// According to Flow, readyState exists on the Document, not the HTMLIFrameElement
+			// Is this different for old IE?
```
Comment (chrislomaxjones) [inline static/src/javascripts/projects/commercial/modules/dfp/get-ad-iframe.ts:17]: Is this comment left over from the old flow javascript, does it need changing it now we've converted to TS, or does it still apply?

**Review by chrislomaxjones — APPROVED** [review]
+1 looks good, just one small question

File: static/src/javascripts/projects/commercial/modules/dfp/get-ad-iframe.ts (hunk 1 above)
Comment (arelra) [reply static/src/javascripts/projects/commercial/modules/dfp/get-ad-iframe.ts:17]: It's still valid: `readyState` on an iframe seems to be an IE thing. I can update it.

**Comment by MarSavar** [comment]
Nice!

**Pull request merged**